CMakeLists.txt

```diff
@@ -368,6 +368,8 @@ if(WITH_RADOSGW)
   endif()
 endif(WITH_RADOSGW)
 
+find_package(liboath REQUIRED)
```

Comment:
i'm guessing that we'll also need to express this dependency in `install-deps.sh` and the rpm/deb builds as well?